cmd/openshift-install/targets.go

```diff
@@ -83,6 +87,10 @@ func runTargetCmd(targets ...asset.WritableAsset) func(cmd *cobra.Command, args
 				return errors.Wrapf(err, "failed to write asset (%s) to disk", a.Name())
 			}
 		}
+		err = assetStore.Save(rootOpts.dir)
```

Comment:
can we save state each time we successfully complete a target?

Reply:
Yes. But we use the same StoreImpl, so it will all get overwritten. And worse, doubly computed.

Reply:
sure, lets keep it as is then.